packages/rancher-cis-benchmark/charts/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: default-clusterscanprofiles
+  namespace: {{ template "cis.namespace" . }}
+data:
+  # Default ClusterScanProfiles per cluster provider type
+  rke: "rke-profile-permissive"
+  eks: "eks-profile"
+  gke: "gke-profile"
+  default: "cis-1.5-profile"
```

Comment:
How is this file used ? I don't see it referenced anywhere.

Reply:
It is referenced by the cis-operator [here](https://github.com/rancher/cis-operator/blob/master/pkg/securityscan/scanHandler.go#L118-L130) - instead of hardcoded default profiles, we decided to maintain it as a configmap in the chart so that if the defaults change we can upgrade the chart.